scripts/S3.R

```diff
@@ -0,0 +1,50 @@
+# create ds object --------------------------------------------------------
+
+ds <- create_dataset(id = "S3")
+
+# query the cube -------------------------------------------------------
+
+ds$query <- '
+PREFIX schema: <http://schema.org/>
+PREFIX cofog: <https://environment.ld.admin.ch/foen/BFS_cofog_national/>
+PREFIX cube: <https://cube.link/>
+
+
+SELECT ?period ?cofog ?sector ?mio_chf_expenditure ?percent_total_expenditure
+WHERE {
+  cofog:1 cube:observationSet ?observationSet0 .
+  ?observationSet0 cube:observation ?source0 .
+  ?source0 cofog:cofog [ schema:name ?cofog ] .
+  ?source0 cofog:period ?period .
+  ?source0 cofog:in-pct ?percent_total_expenditure .
+  ?source0 cofog:in-mio-chf ?mio_chf_expenditure .
+  ?source0 cofog:sector [ schema:name ?sector ] .
+
+  FILTER(LANG(?sector) = "de")
+  FILTER(LANG(?cofog) = "de")
+}
+'
+ds <- download_data(ds)
+
+# data cleaning -----------------------------------------------------------
+
+### clean spatial unit names from artificats
+ds$data %>%
+  janitor::clean_names() %>%
+  dplyr::rename(
+    jahr = period,
+    klassifikation_der_aufgabenbereiche_des_staates = cofog,
+    institutioneller_sektor = sector,
+    ausgaben_in_mio_chf = mio_chf_expenditure,
+    prozent_der_gesamtausgaben = percent_total_expenditure
+  ) -> ds$data
+# add CH as spatial unit -------------------------------
+
+ds$data %>%
+  dplyr::mutate(spatialunit_uid = "0_CH") -> ds$data
```

Comment:
```suggestion
  dplyr::mutate(spatialunit_uid = statbotData::spatial_mapping_country()) -> ds$data
```
just because we have that function. I feel it is cleaner to set the constant at a single place.

Reply:
Makes sense!